pkg/util/constraints/constraints.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2023 ScyllaDB.
+
+package constraints
+
+// Unsigned is a constraint that any unsigned integer type satisfies.
+type Unsigned interface {
+	~uint | ~uint8 | ~uint16 | ~uint32 | ~uint64 | ~uintptr
+}
+
+// Signed is a constraint that any signed integer type satisfies.
+type Signed interface {
```

Comment:
I though it might not have been there when you started.

What's the worst case to happen when they remove it? I assume we'd "fork"/copy that file and maintain it on our own. But if they keep it, we don't have to review, maintain and update our copy, so to me it seems like we can only gain or end up in the same place. They will likely just move it so we only endup adjusting the import path at some point. And we have version pinning and vendoring.

btw. we already use a lot of deps in `v0.x.x` that have the same (non)commitment.